consensus/istanbul/backend/backend.go

```diff
@@ -939,3 +947,16 @@ func (sb *Backend) VerifyValidatorConnectionSetSignature(data []byte, sig []byte
 		return istanbul.CheckValidatorSignature(validator.NewSet(validators), data, sig)
 	}
 }
+
+func (sb *Backend) IsPrimaryForSeq(seq *big.Int) bool {
```

Comment:
suggest we move all functions that are mostly proxies to replicastate togheter (can be on this file o engine... not clear)